terraform/helm/vector-log-agent/values.schema.json

```diff
@@ -1,19 +1,23 @@
 {
   "$schema": "https://json-schema.org/draft-07/schema#",
-  "required": ["k8s_cluster", "logging_sinks", "env"],
+  "required": ["k8sCluster", "loggingSinks", "secretVars"],
```

Comment:
Instead of using an explicit helm values schema file, I've been just defining some sane defaults and thoroughly documenting the values in `values.yaml`, and leaving some empty which are required. Then using `helm-docs` to create a README like so: https://github.com/aptos-labs/aptos-core/blob/main/terraform/helm/aptos-node/README.md#L51